deluge/ui/web/server.py

```diff
@@ -83,6 +83,16 @@ def render(self, request):
         return compress(template.render(), request)
 
 
+class MockGetText(resource.Resource):
+    """Mock the file `gettext.js` in case it does not exists so there will still be
```

Comment:
The first line of a docstring should not continue onto next line and should end with a full-stop. 

This is the style we should be using: http://www.sphinx-doc.org/en/stable/ext/example_google.html